### DISCUSSION TITLE
React Component snapshots - still not supported in v5?

### DISCUSSION THREAD

**Discussion by corinnaSchultz** [avajs/ava#3127] [Q&A] [answered]
In the release notes for V4, there was a statement saying that support for React component snapshots would be moved to a plugin interface.  As far as I can tell this is not yet implemented.  

Did I miss some docs somewhere?  Does anyone know if there's an expected timeline for this availability?

**Comment by novemberborn** [comment, marked as answer]
I made a start, but did not complete it: https://github.com/avajs/ava/pull/2937

There is no timeline. Contributions are welcome, but I appreciate it's undocumented and hard to understand.

**Reply by corinnaSchultz** [reply]
OK, thank you for the information.  No pressure here, I appreciate everything you do :)
